src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -506,6 +540,17 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
         )
     }
 
+    private fun enableProductsAccordingToServerSetting() {
+        pluginSettings().apply {
+            sastOnServerEnabled = service<SnykApiService>().sastOnServerEnabled
+            ossScanEnable = true
+            advisorEnable = sastOnServerEnabled!!
```

Comment:
how advisorEnable and sastOnServerEnabled are connected?

Reply:
see below